src/shim/shim-config.ts

```diff
@@ -168,7 +171,7 @@ class Config implements IConfig {
             ...options.instrumentationOptions,
             http: {
                 ...options.instrumentationOptions?.http,
-                ignoreOutgoingUrls: this.correlationHeaderExcludedDomains, /// TODO: Deprecated configuration
```

Comment:
Created a task to move this logic to the distro using ignoreOutgoingRequests to filter URLs.

Reply:
Looks like correlation headers config is deprecated according to the comment, why you want to move that to distro?

Reply:
We already do processing there in the trace handler for filtering using `ignoredOutgoingRequestHook`. I'm also assuming that if we wanted to block dependency calls to the default excluded domains like `"*.core.windows.net"` in the distro as well it'd make sense to unify that work there. 
However, if we're not planning to filter any domains out in the distro, I can implement this logic here.

Reply:
ignoredOutgoingRequestHook can also be provided by customers or added here in this project, this config is about correlation headers not blocking dependencies, I think you got the concept wrong, current behavior in shim is incorrect

Reply:
I just realized what my confusion here was. I assumed the comment was referring to the `ignoreOutoingUrls` being deprecated (since it is a deprecated config option), while the comment really refers to `correlationHeaderExcludedDomains` being deprecated. Makes far more sense now, thanks!